src/loki/status_action.cc

```diff
@@ -1,8 +1,49 @@
+#include <boost/property_tree/json_parser.hpp>
+#include <iostream>
+#include <sstream>
+
+#include "baldr/tilehierarchy.h"
+#include "config.h"
 #include "loki/worker.h"
+#include "proto/status.pb.h"
 
 namespace valhalla {
 namespace loki {
-void loki_worker_t::status(Api&) const {
+void loki_worker_t::status(Api& request) const {
+
+  auto* status = request.mutable_status();
+
+  // only return more info if explicitly asked for (can be very expensive)
+  // bail if we wont be getting extra info
+  if (!request.options().verbose() || !allow_verbose)
+    return;
+
+  // get _some_ tile
+  static baldr::graph_tile_ptr tile = nullptr;
+  if (!tile) {
+    for (const auto& tile_id : reader->GetTileSet()) {
+      tile = reader->GetGraphTile(tile_id);
+      if (tile->id().level() < baldr::TileHierarchy::GetTransitLevel().level &&
+          tile->header()->nodecount() > 0) {
+        break;
+      }
+    }
+
+    if (connectivity_map) {
+      status->set_bbox(connectivity_map->to_geojson(2));
+    }
+
+    status->set_has_tiles(!!tile);
```

Comment:
didn't know how else to make a boolean out of the tile ptr :D

Reply:
my foggy head isn't really capable of asserting that's correct right now though..